tavla/app/(admin)/components/Login/Create.tsx

```diff
@@ -26,12 +26,23 @@ import { handleError } from 'app/(admin)/utils/handleError'
 import Google from './Google'
 
 function Create() {
+    const [email, setEmail] = useState('')
+    const [password, setPassword] = useState('')
+    const [repeat, setRepeat] = useState('')
     const submit = async (p: TFormFeedback | undefined, data: FormData) => {
```

Comment:
Pirk, kan vi kalle `p` for `previousState` så er det likt med de andre? Evt `prevState`